src/node.cc

```diff
@@ -124,6 +124,7 @@
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
+#include <filesystem>
```

Comment:
I would recommend adding a comment here that `<filesystem>` is imported only to support path-processing operations and not actually performing any file system operations (which should always go through libuv)